### ISSUE TITLE
Move all typos analyzer's parameters to a single config

### ISSUE THREAD

**Issue by irinakhismatullina** [src-d/style-analyzer#685]
Create single config for IdTyposAnalyzer and TyposCorrector models with all default parameters for training and analyzing.

**Comment by EgorBu** [comment]
Even more - some parameters should be shared among several analyzers - `Format`/`Typos`/etc. For example parameters for `filter_files`: 
```python 
"line_length_limit": 500,
"lower_bound_instances": 500,
"overall_size_limit": 5 << 20
```

**Comment by zurk** [comment]
👍 for the issue

However, let's not merge shared parameters for now, because different analyzers will possibly have different default values.

**Comment by EgorBu** [comment]
it's not implemented yet, let's discuss it at ML sync. 
TBH I think some common configuration could be even moved to `lookout/core` (as config for `filter_files` I mentioned above)

**Comment by irinakhismatullina** [comment]
Merged in #718.
